wafer/talks/forms.py

```diff
@@ -25,6 +25,24 @@ def __init__(self, *args, **kwargs):
                             _('Delete')))))
         else:
             self.helper.add_input(submit_button)
+        # insert before authors
+        # TODO: this is *YUK*, but if we add the field to Meta.fields below,
+        # then we can disable the widget, sure, but it'd still load all the 
+        # data, and we'd need to validate that the submitted form isn't trying
+        # to change the data if someone bypasses (removes) the HTML disabled
+        # tag. The following is basically display-only as the input field
+        # has no name.
+        self.helper.layout.insert(TalkForm.Meta.fields.index('authors'),
+                Div(
+                    HTML('<label class="control-label" for="id_corresponding_author">Corresponding author: </label>'),
+                    Div(
+                        HTML('<input id="id_corresponding_author" class="textinput textInput form-control" type="text" value="%s" readonly/>'
+                            % instance.corresponding_author.get_full_name()),
+                        **{'class':'controls'}
```

Comment:
You aren't passing a dict. You're creating a dict, and then passing it as keyword arguments.